src/components/Calculator.jsx

```diff
@@ -0,0 +1,46 @@
+          </ul>
+          <ul className='grid grid-cols-4'>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>7</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>8</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>9</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>*</button></li>
+          </ul>
+          <ul className='grid grid-cols-4'>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>4</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>5</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>6</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>-</button></li>
+          </ul>
+          <ul className='grid grid-cols-4'>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>1</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>2</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>3</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>+</button></li>
+          </ul>
+          <ul className='grid grid-cols-4'>
+            <li className='col-span-2 bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>0</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>.</button></li>
+            <li className='bg-[#e0e0e0] flex justify-center items-center border-4 h-[100%] my-[auto] cursor-pointer'><button className='w-full' type='button'>=</button></li>
+          </ul>
+        </div>
+      </div>
+    )
+  }
+}
+
```

Comment:
- [ ]  Your design should match to the one provide  for you 100%. please make sure that you design resemble to the one from the requirements 

Your design 
![image](https://user-images.githubusercontent.com/43877689/211514188-94217ebd-53bf-4756-969b-4cc9a123175e.png)

Required design 
![image](https://user-images.githubusercontent.com/43877689/211514449-ba1a6400-184f-4841-94f6-34157a250554.png)

Implement the suggested changes as it is suggested in the requirements, so that you can go to the next task.

Reply:
Hii @ishimwezachee 

Thanks for your review. I will work on the suggested changes and complete my project 😊